<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</normaloff>../../../../../Users/User/Documents/Tempовая хуйня/easter egg.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>403</width>
      <height>143</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_task1">
         <property name="text">
          <string>Задание 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_task2">
         <property name="text">
          <string>Задание 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_task3">
         <property name="text">
          <string>Задание 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_task4">
         <property name="text">
          <string>Задание 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new_group">
       <property name="text">
        <string>Добавить новую группу</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_stat">
         <property name="text">
          <string>Статистика</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
